thanks_for_donating_one_time.php

```diff
@@ -33,6 +40,24 @@
           	<p><?php print $donorFirstName; ?>,</p> 
 			<p>We processed your donation for $<?php print $donationAmountDollars; ?> to <?php print $row['project_name']; ?> in <?php print $villageName; ?> Village! You have disrupted 
 			extreme poverty for <?php print $row['peopleCount']; ?> people and <?php print $row['householdCount']; ?> households in <?php print $row['country_label']; ?>.</p>
+			<p class="center-align"><b>Share the good news:</b> 
+						<a
+							href="https://www.facebook.com/dialog/feed?
+  									app_id=<?php print FACEBOOK_APP_ID; ?>
+  									&display=popup&caption=Disrupt extreme poverty by funding projects villages choose.
```

Comment:
spaces in a url is against spec and will probably cause issues somewhere.  We should be using urlencode if we're not already.